qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -52,18 +56,52 @@ class Calibrations:
     ScheduleBlock are supported.
     """
 
+    # The name of the parameter under which the qubit frequencies are registered.
+    __qubit_freq_parameter__ = "qubit_lo_freq"
```

Comment:
I think it is a matter of interpretation and point of view. You can take a model point of view in which this parameter  represents the frequency of the qubit you would put into a Hamiltonian describing the system. My preference for this is then simply `qubit_freq` or `qubit_freq_est` to be pedantic (note also that some architecture do control the qubit freq). Alternatively, you can take the point of view of the control electronics which says this is the frequency of the waveform it will play, e.g. `drive_channel_freq`. This is closer to the hardware reality than `qubit_freq` but it is less close to Hamiltonian. I prefer a language closer to the Hamiltonian.

Reply:
In the lab-frame Hamiltonian view the system will be something like
```
H = w_q a^¥dagger a + w_q Sz/2 + g(a^¥dagger S- + a S+) + a^¥dagger ¥Omega e^(-j w_d t)
```
and `qubit_freq` should refer to `w_q`. I think this parameter should refer to `w_d` (usually this is on-resonant thus `w_q == w_d` and practically no problem). However I think this is really misreading. I've seem several people who believe this controls qubit frequency.

The consistency of `qubit_freq` to `qubit_freq_est` make sense but this is just IBM convention. I think this is agnostic to provider so we should be able to use more general naming convention. @wshanks any thoughts?

Reply:
Indeed _lo_frequency_ is a technical term and if people doesn't know about control electronics they might think this is some frequency of qubit. Then, frequency calibration may looks like we are calibrating qubit frequency, that is not correct interpretation, i.e. we are calibrating the drive channel frequency to math with qubit frequency.

Reply:
Done here 12b3c5f and here 2124fa0